src/afl-fuzz.c

```diff
@@ -1046,9 +1051,9 @@ int main(int argc, char **argv_orig, char **envp) {
 
       }
 
-      show_stats(afl);
+      // show_stats(afl);
```

Comment:
intentional? So show_stats() is unnecessary here?

Reply:
same